indigo_api/static/xsl/fo/_front-matter.xsl

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xsl:stylesheet version="2.0"
+                xmlns:akn="http://docs.oasis-open.org/legaldocml/ns/akn/3.0"
+                xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+  <xsl:template match="akn:frontMatter">
+    <fo:block font-family="{$font-fam-frontmatter}" font-size="{$fontsize-frontmatter-small}">
+      <xsl:apply-templates/>
+    </fo:block>
+  </xsl:template>
+
+  <xsl:template match="akn:container">
```

Comment:
Maybe better placed in blocks.html? When do we use `container`?

Reply:
it's a bit of a hack 🙂